code/modules/integrated_electronics/core/printer.dm

```diff
@@ -262,7 +262,7 @@
 			if("load")
 				if(cloning)
 					return
-				var/input = capped_multiline_input(usr, "Put your code there:", "loading")
+				var/input = capped_multiline_input(usr, "Put your code there:", "loading","", 6000)
```

Comment:
You sure it's not better to set defaul input to null instead of ""?

Reply:
Yeah probably better i did that only so that i have a placeholder in there so i can change the cap size of the string give me a second and i am changing that